sympy/printing/pretty/tests/test_pretty.py

```diff
@@ -5176,3 +5176,10 @@ def test_issue_6134():
      0                              0                   \
 """)
     assert upretty(e) == ucode_str
+
+def test_issue_9877():
+    from sympy.abc import a, b, c, x, y, z
```

Comment:
No need for importing. Already imported on [L28](https://github.com/kshitij10496/sympy/blob/complement_printing/sympy/printing/pretty/tests/test_pretty.py#L28).